src/Google.Storage.V1/StorageClient.ListObjects.cs

```diff
@@ -51,16 +51,15 @@ public abstract partial class StorageClient
         /// </summary>
         /// <remarks>
         /// This method fetches the objects lazily, making requests to the underlying service
-        /// for a page of results at a time, as required. To retrieve all the objects in a single collection,
-        /// simply call LINQ's <c>ToList()</c> method on the returned sequence.
+        /// for a page of results at a time, as required.
         /// </remarks>
         /// <param name="bucket">The bucket to list the objects from. Must not be null.</param>
         /// <param name="prefix">The prefix to match. Only objects with names that start with this string will be returned.
         /// This parameter may be null, in which case no filtering is performed.</param>
         /// <param name="options">The options for the operation. May be null, in which case
         /// defaults will be supplied.</param>
-        /// <returns>A sequence of objects within the bucket.</returns>
-        public virtual IEnumerable<Object> ListObjects(string bucket, string prefix, ListObjectsOptions options = null)
+        /// <returns>An sequence of pages of objects in the specified bucket.</returns>
```

Comment:
Ah, it was just the one - so I did a bit of rebasing, and it's in the right commit now.